src/fsmonitor/linux/inotify_stubs.c

```diff
@@ -58,7 +58,7 @@ static int inotify_return_table[] = {
 
 static void raise_inotify_error(char const *msg)
 {
-        static value *inotify_err = NULL;
```

Comment:
It's not obvious from the diff why declaring a variable not const is a programming error.

Reply:
In this case, the variable is later assigned a "const value *" (returning from caml_named_value). By not declaring const in this declaration, the const attribute is forgotten, which can cause bad surprises later.

Reply:
Thanks for the explanation.  I didn't follow that - I should get in the habit of looking at more context than github shows me.